jetty-runner/src/it/demo-simple-webapp-runner/pom.xml

```diff
@@ -0,0 +1,137 @@
+                  <artifactId>jetty-runner</artifactId>
+                  <version>@project.version@</version>
+                  <type>jar</type>
+                  <overWrite>false</overWrite>
+                  <outputDirectory>${project.build.directory}/</outputDirectory>
+                  <destFileName>jetty-runner.jar</destFileName>
+                </artifactItem>
+                <artifactItem>
+                  <groupId>org.eclipse.jetty.demos</groupId>
+                  <artifactId>demo-simple-webapp</artifactId>
+                  <version>@project.version@</version>
+                  <type>war</type>
+                  <overWrite>false</overWrite>
+                  <outputDirectory>${project.build.directory}</outputDirectory>
+                  <destFileName>demo-simple-webapp.war</destFileName>
+                </artifactItem>
+              </artifactItems>
+              <overWriteReleases>false</overWriteReleases>
+              <overWriteSnapshots>true</overWriteSnapshots>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>@maven.exec.plugin.version@</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>exec</goal>
+            </goals>
+            <phase>generate-test-resources</phase>
+            <configuration>
+              <outputFile>${project.build.directory}/jetty-runner.log</outputFile>
+              <async>true</async>
+              <executable>${java.home}/bin/java</executable>
+              <arguments>
+                <argument>-jar</argument>
+                <argument>${project.build.directory}/jetty-runner.jar</argument>
+                <argument>--out</argument>
+                <argument>${project.build.directory}/jetty-runner.out</argument>
+                <argument>--port</argument>
+                <argument>0</argument>
+                <argument>--port-file</argument>
+                <argument>${project.build.directory}/port.txt</argument>
+                <argument>${project.build.directory}/demo-simple-webapp.war</argument>
+              </arguments>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>@maven.surefire.version@</version>
+        <configuration>
+          <includes>
+            <include>IntegrationTest*.java</include>
+          </includes>
+          <systemPropertyVariables>
+          </systemPropertyVariables>
+          <dependenciesToScan>
+            <dependency>org.eclipse.jetty:jetty-runner</dependency>
+          </dependenciesToScan>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+</project>
```

Comment:
@olamy would any of this test setup be simplified at all if it moved over into tests/test-runner instead?  The integration test setup is impressive, but looks really complex.

Reply:
could be but it will be simply the same poms etc... move to tests/test-runner. 
What if we want to add more tests? I don't want to turn having tests/test-runner-2 etc...

Reply:
This uses the `jetty-runner.jar` as built, and without the maven classloader causing false success.
Does tests/test-runner satisfy these 2 (kinda important) aspects of testing jetty runner?

Reply:
will only move this IT test as a simple project under tests/test-runner. But I don't like it. I prefer this structure using the invoker plugin within jetty-runner project.